### PR TITLE
HTM-1248: Add crosshair cursor for coordinate-link-window and streetview

### PR DESCRIPTION
**Pull request by geertplaisier**
[HTM-1248](https://b3partners.atlassian.net/browse/HTM-1248) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=Tailormap&utm_campaign=badge_info)

[HTM-1248]: https://b3partners.atlassian.net/browse/HTM-1248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/map/src/lib/map/crosshair-cursor.png
Comment (mprins) [inline projects/map/src/lib/map/crosshair-cursor.png:1]: on a high resolution screen this is tiny; much smaller that the regular cursor

File: projects/map/src/lib/map/map.component.css [hunk 1]
```diff
@@ -45,3 +45,8 @@
     font-size: 1.2em;
   }
 }
+
+.map-container--with-crosshair {
+  /* custom icon, default crosshair is a bit thin */
+  cursor: url("crosshair-cursor.png"), crosshair;
```
Comment (mprins) [inline projects/map/src/lib/map/map.component.css:51]: I would prefer having the OS/user provided cursor

**Review by mprins — CHANGES_REQUESTED** [review]
What about "coordinate picker", I think that should get the crosshair as well

**Comment by matthijsln** [comment]
Native cursor looks good on Linux but if on MacOS it's not visible enough there are options to use a bigger icon with scaling: https://stackoverflow.com/questions/51902842/is-it-possible-to-scale-a-custom-cursor-image-using-css

**Review by matthijsln — APPROVED** [review]

**Review by mprins — APPROVED** [review]

**Pull request merged**